resource.address/http/src/main/java/org/kaazing/gateway/resource/address/http/HttpRealmInfo.java

```diff
@@ -24,7 +24,6 @@
 public class HttpRealmInfo {
```

Comment:
Recommend splitting this into a `HttpRealmInfo` interface with getters only and a `DefaultHttpRealmInfo` implementation with setters and getters in an internal package, then mock interface from tests.